src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -33,19 +32,6 @@
     <ProjectReference Include="..\..\System.Runtime.InteropServices.WindowsRuntime\src\System.Runtime.InteropServices.WindowsRuntime.csproj" />
     <ProjectReference Include="..\..\System.ObjectModel\src\System.ObjectModel.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uapaot'">
-    <ReferenceFromRuntime Include="System.Private.Interop" />
-    <ReferenceFromRuntime Include="System.Private.Corelib" />
-    <Reference Include="Windows" />
-    <Reference Include="mscorlib" />
-    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
-      <TargetGroup>uapaot</TargetGroup>
-    </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj">
-      <Aliases>System_Runtime_Extensions</Aliases>
-      <TargetGroup>uapaot</TargetGroup>
-    </ProjectReference>
-  </ItemGroup>
   <ItemGroup Condition="'$(TargetsNETCoreApp)'=='true' or '$(TargetGroup)'=='uap'">
     <Compile Include="System\InternalHelpers.CoreCLR.cs" />
```

Comment:
`.CoreCLR.cs` suffixes on these files do not make sense anymore since we have one and only implementation now. These files can be renamed like `InternalHelpers.CoreCLR.cs` -> `InternalHelpers.cs`, etc.